marklogic-data-hub/src/main/java/com/marklogic/hub/job/JobManager.java

```diff
@@ -30,11 +30,11 @@
 
     /**
      * Creates and returns a JobManager object
-     * @param hubConfig the hubConfig for the JobManager to use
+     * @param jobClient the database client that is used to connect to the jobs database
      * @return JobManager object
      */
-    static JobManager create(HubConfig hubConfig){
-        return new JobManagerImpl(hubConfig);
+    static JobManager create(DatabaseClient jobClient){
+        return new JobManagerImpl(jobClient);
```

Comment:
um, we can't change this in a minor release.

Reply:
This change reverts it back to how it was in 4.0.0